app/src/test/java/com/ikarabulut/GameDisplayTest.java

```diff
@@ -0,0 +1,25 @@
+package com.ikarabulut;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameDisplayTest {
+    @Test
+    @DisplayName("When a new GameDisplay is generated, prompts should not be null")
+    void setPrompts() {
+        GameDisplay gameDisplay = new GameDisplay();
+
+        assertAll (
```

Comment:
I'd add tests here for the actual prompts, but this is fine for now. Just something to note for the future.